standard/conversions.md

```diff
@@ -753,9 +746,27 @@ static void F(double[] a, double[] b) {
 ```
 Since the two anonymous function delegates have the same (empty) set of captured outer variables, and since the anonymous functions are semantically identical, the compiler is permitted to have the delegates refer to the same target method. Indeed, the compiler is permitted to return the very same delegate instance from both anonymous function expressions.
 
-### 11.7.3 Evaluation of anonymous function conversions to expression tree types
+### 11.7.3 Evaluation of lambda expression conversions to expression tree types
+
+Conversion of a lambda expression to an expression tree type produces an expression tree ([§9.6](types.md#96-expression-tree-types)). More precisely, evaluation of the lambda expression conversion produces an object structure that represents the structure of the lambda expression itself.
+
+Not every lambda expression can be converted to expression tree types. The conversion to a compatible delegate type always *exists*, but it may fail at compile-time for implementation-specific reasons.
+
+> *Note*: Common reasons for a lambda expression to fail to convert to an expression tree type include:
+>
+> - It has a block body
+> - It has the `async` modifier
+> - It contains an assignment operator
+> - It contains an `out` or `ref` parameter
+> - It contains a dynamically bound expression
+>
+> *end note*
+
+The following aspects of conversions from lambda expressions to expression trees are implementation-specific:
```

Comment:
This reads to be as out-of-order – the examples in lines 757-761 come _before_ the list of things in lines 767-769 of things which are implantation specific. Surely it should be the other way around?

Reply:
I went back and forth on this. I don't mind whether we go from the detail to the summary or vice versa. Let's talk about it in the meeting.